package.json

```diff
@@ -14,7 +14,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/fwalzel/handlebars-i18n.git"
+    "url": "https://github.com/aller-couleur/handlebars-i18n.git"
```

Comment:
What about using `Aller-Couleur` to not rely on GitHub's case insensitivity?

Reply:
Fine!